libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json

```diff
@@ -2,7 +2,7 @@
     "answers": [
         {
             "score": 68.54820341616869,
-            "Id": 22,
+            "id": 22,
```

Comment:
This is the only change that I'm not sure about

Reply:
Agree, anyway, if we left it as the original version we need to find a workaround for the test that use this file

Reply:
I executed the tests, but it passed correctly.. I'll double check.

Reply:
Yes, they are passing rollbacking the change, it's weird, when I tested it did not pass with the original file

Reply:
I'll confirm this and I'll rollback the change.. if not.. this should be a problem on your side as we are not in parity

Reply:
Fixed